src/components/CavaticaProjectList/CavaticaProjectItem.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+import {List, Icon, Button, Popup, Divider} from 'semantic-ui-react';
+import TimeAgo from 'react-timeago';
+
+const CavaticaProjectItem = ({
+  projectNode,
+  unlinkProject,
+  studyId,
+  disableLink,
+}) => {
+  return (
+    <List.Item>
+      <Icon
+        name={
+          projectNode.projectType === 'DEL'
+            ? 'paper plane outline'
+            : 'sliders horizontal'
+        }
+      />
+      <List.Content>
+        {unlinkProject && (projectNode.study || studyId) ? (
```

Comment:
Could we show the study that it's linked to? That would be helpful to know when we're viewing the list outside a single study's context.